.eleventy.js

```diff
@@ -11,13 +11,10 @@
 */
 "use strict";
 
-var slugify = require("slugify");
-var cleanSlugify = function (string) {
-    return slugify(string, {
-        replacement: "-",
-        lower: true,
-        strict: true
-    });
+var GithubSlugger = require("github-slugger");
+var githubSlugify = function (string) {
+    var slugger = new GithubSlugger();
+    return slugger.slug(string);
 };
```

Comment:
I think you can simplify this down by just using the internal implementation instead of constructing a new instance. While using the constructor is recommended, it appears that the current implementation would create a new instance for each call, likely negating benefits of the single instance. In some local testing, this still produces the desired slugs and increments for duplicate headings. You can look at the "Example:" headers used on DataSourceAPI.md and ConnectingThePartsOfAPreferencesEditor.md. 

You could hoist the `var slugger = new GithubSlugger();` out of the function definition to have a single instance. However, if I'm not mistaken, this would cause duplicate slugs across pages to increment the suffix counter. Which I don't believe we'd want. 

see: https://github.com/Flet/github-slugger/blob/master/README.md

```suggestion
var githubSlugify = require("github-slugger").slug;
```

Reply:
Makes sense, thanks!